src/misclibs.F

```diff
@@ -393,8 +393,11 @@ subroutine convinitu(myid,ib,ie,jb,je,kb,ke,ni,nj,nk,ibw,ibe,   &
       !$acc declare present(xf,yh,zh,u0,u3d)
 
 !!!      if(myid.eq.0) print *,'    convinitu '
+#ifdef _OPENACC
+      print *,'WARNING: OPENACC version of convinitu never verified'
+#endif
       !$omp parallel do default(shared) private(i,j,k,term1,term2,term3,term4,umo)
-      !$acc parallel loop gang vector default(present) private(i,j,k,term1,term2,term3,term4,umo)
+      !$acc parallel loop gang vector collapse(3) default(present)
```

Comment:
I think we have race condition here for both OpenMP and OpenACC here?

Reply:
@sjsprecious, I am not seeing the race condition in this loop.  Could you add more details to your comment?

Reply:
My bad! I thought `term1`, `term2`, `term3`, `term4` and `umo` were returned variables. You are right that there is no race condition here. A good example of not reviewing PR during the midnight.